releasenotes/notes/48312.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+issue:
+- 48349
+releaseNotes:
+- |
+  **Fixed** an issue if DestinationRule's exportTo includes workload's current namespace (not '.'), other namespaces are ignored from ExportTo.
```

Comment:
```suggestion
  **Fixed** an issue where if a `DestinationRule`'s `exportTo` includes the workload's current namespace (not '.'), other namespaces are ignored from `exportTo`.
```

Reply:
@hanxiaop is this going to cause issues with being out of sync with master branch?

Reply:
This is a good question. I think that regardless of this issue, this kind of release notes will be modified by RMs or doc maintainers after the release note PR is submitted to the doc site. Let's just merge it right now and make modifications later.

Reply:
BTW I know there's a situation where a fix was merged without notes, and the RM required a note. Eventually, the release note only existed in the release branch.